### DISCUSSION TITLE
DockerDriver - container running for a few seconds

### DISCUSSION THREAD

**Discussion by idan4u** [labgrid-project/labgrid#1408] [Q&A]
Hello!

I'm trying to use labgrid with DockerDaemon + DockerDriver.

When I'm using the following command:

labgrid-client -c ../first_test/local.yaml power on

I can see for one second the new container but then it just disappear (by using docker ps command)

this is my local.yaml config file:

targets:
  main:
    resources:
      RemotePlace:
        name: idan
    drivers:
      DockerDriver:
        image_uri: 'bash:devel-alpine3.19'
        container_name: 'ubuntu-lg-example'
        host_config: {'network_mode': 'bridge'}
        network_services: [{'port': 22, 'username': 'root', 'password': 'root'}]
        command: 'sleep 500'
      DockerStrategy: {}

Thank you for your help.

**Comment by Emantor** [comment]
This might need further debugging of the docker driver, it was an external contribution and I unfortunately don't have much expertise with the driver.

**Reply by idan4u** [reply]
Thank you!

**Reply by jluebbe** [reply]
It was merged via https://github.com/labgrid-project/labgrid/pull/301. @dutsii, do you have some spare time to help?
